.dprint.jsonc

```diff
@@ -0,0 +1,42 @@
+{
+  "excludes": [
+    // These files are auto-generated and don't need to be formatted.
+    // NOTE: We generally prefer that packages be formatted with dprint.
+    // Packages are generally only listed if they predate our using dprint (August 2023).
+    // If your package is listed, please consider adding dprint fmt to its generation.
+    "./types/custom-functions-runtime",
+    "./types/dojo",
+    "./types/facebook-nodejs-business-sdk",
+    "./types/fhir",
+    "./types/fibjs",
+    "./types/get-intrinsic",
+    "./types/locutus",
+    "./types/lodash",
+    "./types/material-ui",
+    "./types/meteor",
+    "./types/microsoft-graph",
+    "./types/nginstack__engine",
+    "./types/node",
```

Comment:
Can we be more specific for the exclusion on this one? IIRC it's just specific files; potentially they could have explicit dprint-ignore-files or something too.

Reply:
Hmm, do you have a contact for someone who'd know for sure? I'm having a hard time determining which ones are auto-generated from just what's on disk or in PR history.

cc @peterblazejewicz based on recent PRs?

Reply:
The file I'm thinking of is `inspector.d.ts` as it has an associated script in `scripts`.

Reply:
Makes sense. I'll limit to just that.